packages/docs/src/routes/docs/overview/index.mdx

```diff
@@ -22,7 +22,7 @@ Qwik is a new kind of web framework that can deliver instant loading web applica
 <div class="card-grid">
   <a class="card card-center" href="/docs/getting-started/">
     <p class="icon">🚀</p>
-    <h3>npm create qwik</h3>
+    <h3>Getting Started</h3>
```

Comment:
not sure about this one. it's everywhere else written as `Getting Started` and thought about the npm -> pnpm transition you might want to push npm a bit less 👼 let me know what is preferred 🙏 
cc: @shairez

Reply:
I think it's find to call it "getting started"

If anyone will have issues with it, we can always revert this one line, don't want to postpone the entire PR because of this one title